src/pages/CommitDetailPage/CommitCoverage/CommitCoverage.jsx

```diff
@@ -153,17 +152,23 @@ function CommitCoverageRoutes() {
 function CommitCoverage() {
   const { provider, owner, repo, commit: commitSha } = useParams()
   const { data: tierName } = useTier({ owner, provider })
-  const { data: repoData } = useRepoSettingsTeam()
+  const { data: overview } = useRepoOverview({ provider, owner, repo })
   const { data: commitPageData } = useCommitPageData({
     provider,
     owner,
     repo,
     commitId: commitSha,
   })
 
-  const showCommitSummary = !(
-    repoData?.repository?.private && tierName === TierNames.TEAM
-  )
+  useEffect(() => {
+    if (overview.bundleAnalysisEnabled && overview.coverageEnabled) {
+      metrics.increment('commit_detail_page.coverage_dropdown.opened', 1)
+    } else if (overview.coverageEnabled) {
+      metrics.increment('commit_detail_page.coverage_page.visited_page', 1)
```

Comment:
Sentry metrics increments by 1 by default. Maybe we can do the same in our abstraction?

Reply:
I'd rather leave the wrapper to be as close to the Sentry metric implementation, incase anyone reads their docs it will match what they're reading (minus the limited set of names)